.editorconfig

```diff
@@ -9,3 +9,6 @@ indent_style = space
 endofline = lf
 trim_trailing_whitespace = true
 insert_final_newline = true
+
+[*.json]
+indent_size = 2
```

Comment:
Does not matter for me. My editor setting is reformat the edited file when save. I edited package.json file and the editor replaced all 2 space to 4. 
I thought better if add this setting instead of reformat the package.json file.

I could delete this setting and save the json file in the correct format